add discord verification, how to delete and how to edit a domain (#18)
is-a-dev/docs: src/docs/delete_domain/index.md

# How to delete your is-a.dev domain

## Open your fork of the [is-a-dev/register](https://github.com/is-a-dev/register) repository

> 1. Open your fork, or if you deleted it, [fork](https://github.com/is-a-dev/register/fork) again the repository

Reviewer: ```suggestion
1. Open your fork, or if you deleted it, [fork](https://github.com/is-a-dev/register/fork) the repository once again.
```

> 2. Find you domain file in the `domains` folder (`/domains/<subdomain>.json`)

Reviewer: ```suggestion
2. Find your domain file in the `domains/` folder (e.g. `/domains/myname.json`)
```

> 3. Press the 3 dots and press the "Delete file" button

Reviewer: ```suggestion
3. Press the 3 dots and press the `Delete file` button.
```

<img src="../img/delete_domain_step_1.png" height="250">

> 4. Press the button "Commit changes"

Reviewer: ```suggestion
4. Press `Commit changes`.
```

<img src="../img/delete_domain_step_2.png" height="250">

> 5. In the popup that appears, press the button "Commit changes"

Reviewer: ```suggestion
5. In the popup that appears, press `Commit changes` once again.
```

<img src="../img/delete_domain_step_3.png" height="250">

> 6. Press the "Contribute" button and then press on the "Open pull request" button

Reviewer: ```suggestion
6. Click the `Contribute` button, then press `Open pull request`.
```

<img src="../img/delete_domain_step_4.png" height="250">

> 7. Press the "Create pull request" button

Reviewer: ```suggestion
7. Press `Create pull request` again.
```

<img src="../img/delete_domain_step_5.png" height="250">

> And you're done! After you pull request has been merged, the domain will be deleted

Reviewer: ```suggestion
And you're done! After you pull request has been merged, the domain will be deleted.
```
